Build fails on macOS Big Sur

I know it is still not a released OS version so probably fine to ignore this for now. Anyway, on current `trunk`:

```
$ make clean && make all
git submodule init
git submodule update
mkdir -p target/release
cd librubyfmt && cargo build --release
   Compiling rubyfmt v0.1.0 (/Users/mikker/dev/vendor/rubyfmt/librubyfmt)
The following warnings were emitted during compilation:

warning: In file included from src/rubyfmt.c:1:
warning: In file included from /Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//include/ruby.h:33:
warning: /Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//include/ruby/ruby.h:24:10: fatal error: 'ruby/config.h' file not found
warning: #include "ruby/config.h"
warning:          ^~~~~~~~~~~~~~~
warning: /Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//include/ruby/ruby.h:24:10: note: did not find header 'config.h' in framework 'ruby' (loaded from '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks')
warning: 1 error generated.

error: failed to run custom build command for `rubyfmt v0.1.0 (/Users/mikker/dev/vendor/rubyfmt/librubyfmt)`

Caused by:
  process didn't exit successfully: `/Users/mikker/dev/vendor/rubyfmt/librubyfmt/target/release/build/rubyfmt-63023300acce2427/build-script-build` (exit code: 1)
  --- stdout
  TARGET = Some("x86_64-apple-darwin")
  OPT_LEVEL = Some("3")
  HOST = Some("x86_64-apple-darwin")
  CC_x86_64-apple-darwin = None
  CC_x86_64_apple_darwin = None
  HOST_CC = None
  CC = None
  CFLAGS_x86_64-apple-darwin = None
  CFLAGS_x86_64_apple_darwin = None
  HOST_CFLAGS = None
  CFLAGS = None
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some("false")
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2,sse3,ssse3")
  running: "cc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "/Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//include" "-I" "/Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//.ext/include/x86_64-darwin19" "-I" "/Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//.ext/include/x86_64-darwin18" "-I" "/Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//.ext/include/x86_64-linux" "-Wall" "-Wextra" "-o" "/Users/mikker/dev/vendor/rubyfmt/librubyfmt/target/release/build/rubyfmt-86159fab974d1d4e/out/src/rubyfmt.o" "-c" "src/rubyfmt.c"
  cargo:warning=In file included from src/rubyfmt.c:1:
  cargo:warning=In file included from /Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//include/ruby.h:33:
  cargo:warning=/Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//include/ruby/ruby.h:24:10: fatal error: 'ruby/config.h' file not found
  cargo:warning=#include "ruby/config.h"
  cargo:warning=         ^~~~~~~~~~~~~~~
  cargo:warning=/Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//include/ruby/ruby.h:24:10: note: did not find header 'config.h' in framework 'ruby' (loaded from '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks')
  cargo:warning=1 error generated.
  exit code: 1

  --- stderr


  error occurred: Command "cc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "/Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//include" "-I" "/Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//.ext/include/x86_64-darwin19" "-I" "/Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//.ext/include/x86_64-darwin18" "-I" "/Users/mikker/dev/vendor/rubyfmt/librubyfmt/ruby_checkout/ruby-2.6.6//.ext/include/x86_64-linux" "-Wall" "-Wextra" "-o" "/Users/mikker/dev/vendor/rubyfmt/librubyfmt/target/release/build/rubyfmt-86159fab974d1d4e/out/src/rubyfmt.o" "-c" "src/rubyfmt.c" with args "cc" did not execute successfully (status code exit code: 1).


make: *** [target/release/librubyfmt.a] Error 101
```

I have tried to change where cc looks for the Ruby headers but wasn't successful.

--- a/librubyfmt/build.rs
+++ b/librubyfmt/build.rs
@@ -44,6 +44,10 @@ fn main() {
         .file("src/rubyfmt.c")
         .include(format!("{}/include", ruby_checkout_path.display()))
         .include(format!(
+            "{}/.ext/include/x86_64-darwin20",
+            ruby_checkout_path.display()
+        ))
+        .include(format!(
             "{}/.ext/include/x86_64-darwin19",
             ruby_checkout_path.display()
         ))

